test/metric/active_record_test.rb

```diff
@@ -237,6 +250,47 @@ class Sky < ActiveRecord::Base
     assert_equal 2, times
   end
 
+  it "with model identity" do
+    File.open "tmp/experiments/metrics/sky_is_limit.rb", "w" do |f|
+      f.write <<-RUBY
+        metric "Sky is limit" do
+          model Sky, :identity => lambda { |record| record.user_id }
+        end
+      RUBY
+    end
+
+    File.open "tmp/experiments/simple.rb", "w" do |f|
+      f.write <<-RUBY
+        ab_test "simple" do
+          metrics :sky_is_limit
+          alternatives :a, :b
+          identity { "me" }
+        end
+      RUBY
+    end
+
+    user = User.create
+    Vanity.context = stub( vanity_identity: user.id )
+    experiment = Vanity.playground.experiment(:simple)
+    experiment.choose
+
+    Vanity.context = nil
+    Sky.create do |sky|
+      sky.user_id = user.id
+    end
+    Sky.create do |sky|
+      sky.user_id = user.id + 1
+    end
+
+    Vanity.context = stub( vanity_identity: "other" )
```

Comment:
Please remove extra spaces around parens in these 2 lines to maintain consistency.